darts/models/forecasting/multivariate_forecasting_model_wrapper.py

```diff
@@ -0,0 +1,151 @@
+        return self._predict(n, future_covariates, num_samples, **kwargs)
+
+    def _predict(
+        self,
+        n: int,
+        future_covariates: Optional[TimeSeries] = None,
+        num_samples: int = 1,
+        verbose: bool = False,
+        **kwargs,
+    ) -> TimeSeries:
+        predictions = [
+            model.predict(n=n, future_covariates=future_covariates)
+            if isinstance(model, FutureCovariatesLocalForecastingModel)
+            else model.predict(n=n)
+            for model in self._trained_models
+        ]
+
+        multivariate_series = concatenate(predictions, axis=1)
+        return multivariate_series
+
+    def __init__(self, model: LocalForecastingModel):
+        """
+        Wrapper around LocalForecastingModel allowing it to act like a GlobalForecastingModel
+
+        A copy of the provided model will be trained on each of the components of the provided series separately.
+        The model doesn't use series supplied during predict() and instead predicts on the series it trained on.
+
+        Parameters
+        ----------
+        model
+            Model used to predict individual components
+        """
+        super().__init__()
+
+        self.base_model: LocalForecastingModel = model
+        self._trained_models: List[LocalForecastingModel] = []
+
+    def _split_multivariate(self, time_series: TimeSeries):
+        """split multivariate TimeSeries into a list of univariate TimeSeries"""
+        return [
+            time_series.univariate_component(i) for i in range(time_series.n_components)
+        ]
+
+    @property
+    def extreme_lags(
+        self,
+    ) -> Tuple[
+        Optional[int],
+        Optional[int],
+        Optional[int],
+        Optional[int],
+        Optional[int],
+        Optional[int],
+    ]:
+        return self.base_model.extreme_lags
+
+    @property
+    def _model_encoder_settings(
+        self,
+    ) -> Tuple[
+        Optional[int],
+        Optional[int],
+        bool,
+        bool,
+        Optional[List[int]],
+        Optional[List[int]],
+    ]:
+        return None, None, False, self.supports_future_covariates, None, None
+
+    @property
+    def supports_multivariate(self) -> bool:
+        return True
+
+    @property
+    def supports_past_covariates(self) -> bool:
+        return False
+
+    @property
+    def supports_future_covariates(self) -> bool:
+        return self.base_model.supports_future_covariates
+
+    @property
+    def supports_static_covariates(self) -> bool:
+        return False
+
+    def _is_probabilistic(self) -> bool:
+        """
+        A GlobalForecastingModelWrappers is probabilistic if the base_model
+        is probabilistic
+        """
+        return self.base_model._is_probabilistic()
+
+    def _supports_non_retrainable_historical_forecasts(self) -> bool:
+        return isinstance(
+            self.base_model, TransferableFutureCovariatesLocalForecastingModel
```

Comment:
why not just using the `base_model` property?

Reply:
I considered it, but I wasn't sure how rigid Darts code style is when it came to accessing private variables, so I decided to err on the side of caution and check it manually. That said, if it's not a problem I'd lean towards your suggestions which is significantly more readable